fossology/FOSSologyUI
# feat(UI): Import CSV-license file PAGE

Signed-off-by: dushimsam <dushsam@gmail.com>

## Description

Added a new page for handling the upload of the CSV-license file.

### Changes

1. Added a new file [src/pages/Admin/Group/License/Import/index.jsx]
3. Introduced a new route in `src/Routes.jsx`
4. Introduced new API callers for the UI integration.

## How to test

1. Go to the project's main directory.
2. run command `npm start
3. Visit the page at `http://localhost:PORT/admin/license/import` 
4. The page should look as follows

![image](https://user-images.githubusercontent.com/66276301/188610009-73d56e61-e470-4f7c-b50f-10379f930752.png)

## Related Issue 
Closing #257

cc: @shaheemazmalmmd @GMishx
